internal/oidc/login/post_login_handler_test.go

```diff
@@ -410,6 +410,31 @@ func TestPostLoginEndpoint(t *testing.T) {
 			wantDownstreamPKCEChallengeMethod: downstreamPKCEChallengeMethod,
 			wantDownstreamCustomSessionData:   expectedHappyLDAPUpstreamCustomSession,
 		},
+		{
+			name: "happy LDAP login when groups scope is not requested",
```

Comment:
Hm with LDAP/AD this means we fetch groups but throw them away?  Seems like something we may want to fix in a followup?